multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -140,6 +173,18 @@ func (r *MemberClusterAnnounceReconciler) Reconcile(ctx context.Context, req ctr
 		// Nothing to handle in this case and MemberClusterAnnounce will also be deleted soon.
```

Comment:
@luolanzone : could you revisit this statement? I feel it is not valid.

Reply:
@luolanzone : could you help check all these comments? I do not understand what does "If err != nil, probably ClusterClaims were deleted during the processing of MemberClusterAnnounce." mean, could you clarify?

At least we should move them to be after line 140.

Reply:
Sure, I will check, I was trying to understand this comment but @aravindakidambi  probably is the best person to clarify the comments on common area. not sure If she has the time or not, I will double check with her if I can't understand the background and comments either.

Reply:
@hjiajing : please move the comments to be after line 142 (line 137 in the old code) for now.